screens/PointsHistoryScreen.js

```diff
@@ -0,0 +1,58 @@
+import * as Font from 'expo-font';
+import React from 'react';
+import { AsyncStorage, ScrollView, Text, View } from 'react-native';
+import getCustomerTransactions from './historyHelpers';
+
+import Transactions from '../components/Transactions';
+
+export default class PointsHistoryScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      transactions: [],
+      recent: []
+    };
+  }
+
+  async componentDidMount() {
+    const userId = await AsyncStorage.getItem('userId');
+    getCustomerTransactions(userId).then(transactions => {
+      const recent = [];
+      recent.push(transactions.shift());
+      recent.push(transactions.shift());
+      recent.push(transactions.shift());
```

Comment:
Probably a little redundant to call `.shift()` 3x here, I'd recommend using the splice method! Shift is also memory intensive because it has to move the move every item in the array over

`const recent = transactions.splice(0, 3).reverse();` should achieve the same thing you want in!